methods/guest/src/bin/method_name.rs

```diff
@@ -1,7 +1,6 @@
 // TODO: Rename this file to change the name of this method from METHOD_NAME
 
 #![no_main]
-#![no_std]
```

Comment:
I think we should keep `#![no_std]` as an example since `std` support is experimental and is known to have some issues.

Reply:
Makes sense, will restore